src/test/kotlin/com/noumenadigital/npl/cli/BinaryCommandsIT.kt

```diff
@@ -31,6 +31,8 @@ class BinaryCommandsIT :
                     help       Display the description for npl-cli commands
                     check      Validate the correctness of NPL sources
                                  <directory>  Target directory containing NPL source files to check (defaults to current directory)
+                    openapi    Generate the openapi specifications of NPL api
+                                 <directory>  Target directory containing NPL protocols (defaults to current directory)
                     """.trimIndent()
 
```

Comment:
thinking about it, "Target directory" should probably be "Source directory" for both `openapi` and `check`